region/maridia/outer/Mt. Everest.json

```diff
@@ -989,12 +996,20 @@
       "name": "Suitless Frozen Crab",
       "requires": [
         "canSuitlessMaridia",
-        "canTrickyUseFrozenEnemies"
+        "canTrickyUseFrozenEnemies",
+        {"or": [
+          {"and": [
+            "canTrickyJump",
+            "h_canCrouchJumpDownGrab"
+          ]},
+          "canSpringBallJumpMidAir",
+          "HiJump"
```

Comment:
Same here as 2->7->8 strats. The spring ball strat also has slightly more requirements in that case as well.